2001-3000/LIT2192Profit.xml

```diff
@@ -35,17 +34,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+         <abstract>
+            <p>Theological treatise of the Talmid-type, translated probably  
+               <date notBefore="1475" notAfter="1499" cert="medium">towards the end of the 15th century</date> from a larger work of the
+               Arab theologian <persName ref="PRS14741GirgisY">Ǧirǧis al-Makīn ibn al-ʿAmīd (the Younger)</persName>, who is
+               called "the profitable compiler" in the treatise.</p>
```

Comment:
If "the profitable compiler" is a title, maybe each word could be capitalized. But if it's not, then it is perfect as it is